content/20.pharmaceuticals.md

```diff
@@ -85,86 +87,78 @@ The rapid timescale of this response meant that, initially, most evidence came f
-Therapeutics to treat the most severe cases of COVID-19 are therefore urgently needed, and identifying strategies for treating patients is a priority for clinicians.
+Therapeutics to treat severe cases of COVID-19 are urgently needed, and identifying strategies for treating patients is a priority for clinicians.
 
 In the early days of the COVID-19 pandemic, physicians sought to identify potential treatments that could benefit patients, and in some cases shared their experiences and advice with the medical community on social media sites such as Twitter [@doi:10.7759/cureus.13594].
-These on-the-ground treatment strategies could then be analyzed retrospectively in observational studies or investigated in an interventional paradigm through randomized control studies.
+These on-the-ground treatment strategies could later be analyzed retrospectively in observational studies or investigated in an interventional paradigm through randomized control studies.
 Several notable cases involved the use of small-molecule drugs, which are synthesized compounds of low molecular weight, typically less than 1 kilodalton (kDa) [@url:https://www.nuventra.com/resources/blog/small-molecules-versus-biologics/].
 Small-molecule pharmaceutical agents have been a backbone of drug development since the discovery of penicillin in the early twentieth century [@doi:10.1126/science.287.5460.1960].
 It and other antibiotics have long been among the best known applications of small molecules to therapeutics, but biotechnological developments such as the prediction of protein-protein interactions have facilitated advances in precise targeting of specific structures using small molecules [@doi:10.1126/science.287.5460.1960].
 Small molecule drugs today encompass a wide range of therapeutics beyond antibiotics, including antivirals, protein inhibitors, and many broad-spectrum pharmaceuticals.
-Broad-spectrum treatments are not specifically targeted at a virus or at particular host receptors, but rather induce broad shifts in host biology that are hypothesized to be potential inhibitors of the virus.
-Instead, when a virus enters a host, the host becomes the virus's environment, and, therefore, the state of the host can also influence the virus's ability to replicate and spread.
-The administration and assessment of such treatments on a rapid time course was feasible because they are often either available in hospitals, or in some cases may also be prescribed to a large number of out-patients.
-One of the other advantages is that these well-established compounds, if found to be beneficial, tend to be more widely available than boutique experimental drugs.
+
+Many treatments considered for COVID-19 have relied on a broad-spectrum approach.
+Such treatments do not specifically target a virus or particular host receptor, but rather induce broad shifts in host biology that are hypothesized to be potential inhibitors of the virus.
```

Comment:
Is this specifically in the antiviral space (as written, I think it is)? It feels like the immunomodulatory treatments (especially steroids) are the quintessential broad-spectrum treatment.

Reply:
Good point! Something definitely got lost here in the rearranging.
```suggestion
In addition to broad-spectrum treatments like dexamethasone, antivirals offer an alternative approach.
These treatments do not specifically target a virus or particular host receptor, but rather induce broad shifts in host biology that are hypothesized to be potential inhibitors of the virus.
```

Reply:
👍